scripts/build/artifactory.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
I guess this is related to [general relfections](https://github.com/CeON/rJavaPackageExample/pull/6#issuecomment-185731696) comment and it should be discussed together with it.